examples/intermediate.jl

```diff
@@ -0,0 +1,141 @@
+@block KimFung [intermediate] begin
+
+    @cell "simple line plot" [lines] begin
+        scene = Scene()
+        lines!(scene, rand(10), linewidth=3.0, color=:blue)
+        lines!(scene, rand(10), linewidth=3.0, color=:red)
+        lines!(scene, rand(10), linewidth=3.0, color=:green)
+        lines!(scene, rand(10), linewidth=3.0, color=:purple)
+        lines!(scene, rand(10), linewidth=3.0, color=:orange)
+        display(scene)
+    end
+
+    @cell "multiple functions" [lines] begin
+        scene = Scene()
+        x = range(0, stop = 3pi, step = 0.01)
+        lines!(scene, x, sin.(x), color = :black)
+        lines!(scene, x, cos.(x), color = :blue)
+        display(scene)
+    end
+
+    @cell "animations" [lines] begin
+        scene = Scene()
+        mytime = Node(0.0)
+        f(v, t) = sin(v + t)
+        g(v, t) = cos(v + t)
+        lines!(scene,lift(t -> f.(range(0, stop = 2pi, length = 50), t), mytime),color = :blue)
+        lines!(scene,lift(t -> g.(range(0, stop = 2pi, length = 50), t), mytime),color = :orange)
+        record(scene, "mygraph.gif", range(0, stop = 4pi, length = 100)) do i
+            mytime[] = i
+        end
+    end
+
+    @cell "parametric plots" [poly] begin
+        x = LinRange(0, 2pi, 100)
+        poly(Point2f0.(zip(sin.(x), sin.(2x))), color = "orange", strokecolor = "blue", strokewidth = 4)
+    end
+
+    @cell "colors" [lines] begin
+        lines(1:50, 1:1, linewidth = 20, color = to_colormap(:viridis, 50))
+    end
+
+    @cell "image" [image] begin
+        import FileIO
+        img = FileIO.load(download("http://makie.juliaplots.org/dev/assets/logo.png"))
+        image(img, scale_plot = false)
+    end
+
+    @cell "attributes" [attributes, lines, color, colormap, linewidth] begin
+        scene = Scene()
+        lines!(scene, rand(10), color = to_colormap(:viridis, 10), linewidth = 5)
+        lines!(scene, rand(20), color = :red, alpha = 0.5)
+        display(scene)
+    end
+
+    @cell "build plot in pieces" [lines] begin
```

Comment:
Better to use a [Stepper](http://makie.juliaplots.org/dev/output.html#Stepper-plots-1) here

Reply:
Done!